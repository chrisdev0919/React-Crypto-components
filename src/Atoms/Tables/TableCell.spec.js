import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

import TableCell from './TableCell'

describe('TableCell component', () => {
  it('renders correctly', () => {
    const component = shallow(<TableCell>MyTableCell</TableCell>)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
