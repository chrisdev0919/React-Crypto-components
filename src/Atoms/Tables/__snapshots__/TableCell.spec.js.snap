// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableCell component renders correctly 1`] = `
<styled.div>
  MyTableCell
</styled.div>
`;
